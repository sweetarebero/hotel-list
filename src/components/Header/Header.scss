.Header {
  .container-fluid {
    padding: 20px 0;
  }
  img {
    width: 200px;
  }
}
