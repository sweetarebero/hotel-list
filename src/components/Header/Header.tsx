import * as React from "react";
import Logo from "../../images/qantas-logo.png";
import { Navbar, NavbarBrand } from "reactstrap";

import "./Header.scss";

interface Props {}

const Header = (props: Props) => {
  return (
    <header className="Header">
      <Navbar>
        <NavbarBrand href="/">
          <img src={Logo} alt="Qantas logo" title="go to home page" />
        </NavbarBrand>
      </Navbar>
    </header>
  );
};

export default Header;
