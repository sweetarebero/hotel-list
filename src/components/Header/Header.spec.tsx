import React from "react";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import Header from "./Header";

describe("Header", () => {
  it("renders correctly", () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
