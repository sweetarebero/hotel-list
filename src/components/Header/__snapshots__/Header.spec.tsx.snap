// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div>
  <header
    class="Header"
  >
    <nav
      class="navbar"
    >
      <div
        class="container-fluid"
      >
        <a
          class="navbar-brand"
          href="/"
        >
          <img
            alt="Qantas logo"
            src="qantas-logo.png"
            title="go to home page"
          />
        </a>
      </div>
    </nav>
  </header>
</div>
`;
